codegen/model/chart.go

```diff
@@ -12,6 +12,22 @@ import (
 )
 
 // Chart provides the input data structure for generating Helm charts from the skv2 chart "meta-templates" (templates whose outputs are templates and other files used by generated Helm charts)
+const (
+	CRDVersionKey  = "crd.solo.io/version"
+	CRDSpecHashKey = "crd.solo.io/specHash"
+	CRDMetadataKey = "crd.solo.io/crdMetadata"
+)
+
+type CRDMetadata struct {
```

Comment:
nit: keep nested types below top-level types

Reply:
CRDMetadata is not a nested type?

Reply:
i see. i was confused because `chart.go` contains the types that are used to provide a model for how the chart should be generated. do you think it makes sense to expose these structs from the same package? also it seems like this is code could be used outside of code generation